test/docs/README.md

```diff
@@ -5,6 +5,7 @@
 | [High-Level Description (HLD) Test Specification](dash-test-HLD.md) | High-level design for the testing of devices which conform to the SONiC-DASH requirements.|  
 | [Dash Test Maturity Stages](dash-test-maturity-stages.md) | Describes a progressive approach to DASH testing.|  
 | [DASH SAI-Thrift Test Workflow](dash-test-workflow-saithrift.md) | DASH test workflow with SAI-thrift. |
+| [SAI PTF Design](https://github.com/reshmaintel/SAI/blob/dash-ptf/doc/SAI-Proposal-SAI-PTF.md) | SAI Thrift auto-generated Python based testing framework doc. |
```

Comment:
Please use upstream URL: https://github.com/reshmaintel/SAI/blob/dash-ptf/doc/SAI-Proposal-SAI-PTF.md

Reply:
@chrispsommers the modified design doc is not upstreamed yet, so based on https://github.com/Azure/DASH/pull/171#discussion_r937058503 comment, I've added dev branch for now. Later it can be updated to up streamed one.